master/internal/config.go

```diff
@@ -92,10 +92,7 @@ type Config struct {
 	Logging               model.LoggingConfig               `json:"logging"`
 	HPImportance          hpimportance.HPImportanceConfig   `json:"hyperparameter_importance"`
 
-	Scheduler   *resourcemanagers.Config `json:"scheduler"`
-	Provisioner *provisioner.Config      `json:"provisioner"`
-	*resourcemanagers.ResourcePoolsConfig
-	ResourceManager *resourcemanagers.ResourceManagerConfig `json:"resource_manager"`
+	*resourcemanagers.ResourceConfig
```

Comment:
@shiyuann Combining resource pool and resource manager configs into a single ResourceConfig is just a style change, right? Or is there a technical reason to do that?